pkgs/development/python-modules/rx/default.nix

```diff
@@ -20,12 +23,7 @@ buildPythonPackage rec {
     sha256 = "b657ca2b45aa485da2f7dcfd09fac2e554f7ac51ff3c2f8f2ff962ecd963d91c";
   };
 
-  nativeCheckInputs = [ nose ];
-
-  # Some tests are nondeterministic. (`grep sleep -r tests`)
-  # test_timeout_schedule_action_cancel: https://hydra.nixos.org/build/74954646
-  # test_new_thread_scheduler_timeout: https://hydra.nixos.org/build/74949851
-  doCheck = false;
+  doCheck = false; # PyPI tarball does not provides tests
```

Comment:
Why not fetch from git then?

Reply:
There is a notice about it above:
```
  # Use fetchPypi to avoid the updater script to migrate it to `reactivex` which
  # is being developed in the same repository

```

Reply:
The branch `release/v3.2.x` via `fetchGitHub` should be used as it still only `rx` with tests and not `reactivex`

Reply:
I think this is outside the scope of this pr isn't it?